src/main/java/com/ajou/hertz/domain/user/controller/UserController.java

```diff
@@ -117,6 +119,24 @@ public ResponseEntity<UserResponse> signUpV1(
 			.body(UserResponse.from(userCreated));
 	}
 
+	@Operation(
+		summary = "프로필 이미지 변경",
+		description = "프로필 이미지를 변경합니다.",
+		security = @SecurityRequirement(name = "access-token")
+	)
+	@PutMapping(
+		value = "/me/profile-image",
```

Comment:
```suggestion
		value = "/me/profile-images",
```

컨벤션 맞춰 복수형으로 작성해주세요~